rotkehlchen/globaldb/upgrades/v5_v6.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from rotkehlchen.globaldb.cache import (
+    GENERAL_CACHE_TABLE_NAME,
+    UNIQUE_CACHE_KEYS,
+    UNIQUE_CACHE_TABLE_NAME,
+)
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.drivers.gevent import DBConnection, DBCursor
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+def _create_new_tables(cursor: 'DBCursor') -> None:
+    log.debug('Enter _create_new_tables')
+    cursor.execute(
+        """
+        CREATE TABLE IF NOT EXISTS unique_cache (
+            key TEXT NOT NULL PRIMARY KEY,
+            value TEXT NOT NULL,
+            last_queried_ts INTEGER NOT NULL
+        );
+        """,
+    )
+    log.debug('Exit _create_new_tables')
+
+
+def _transfer_unique_cache_data(cursor: 'DBCursor') -> None:
+    log.debug('Enter _transfer_unique_cache_data')
+    for key_part in UNIQUE_CACHE_KEYS:
+        cursor.execute(
+            f'INSERT INTO {UNIQUE_CACHE_TABLE_NAME} '
```

Comment:
It should not be the case, but just in case we accidentally got multiple values for some of the caches with unique key, I would adjust the query to use `INSERT OR IGNORE`, so that the upgrade doesn't break